contracts/protocol/src/index.ts

```diff
@@ -1,3 +1,2 @@
 export * from './artifacts';
 export * from './wrappers';
-export * from '../test/utils';
```

Comment:
This line was causing a failure in `test-publish`. The files in `../test/utils` import the `@0x/contracts-test-utils` package. Since `@0x/contracts-test-utils` is listed in `devDependencies` and not `dependencies` this results in an error when you try to import the `@0x/contracts-protocol` package.